src/content/getting-started/designers/designers.mdx

```diff
@@ -2,120 +2,179 @@
+3. **You're all set**. Refer to the content below for a more detailed breakdown of the kit, or head to Sketch to begin designing.
 
-There are three ways to obtain the kit; via Sketch shared library, via Box Drive, and via GitHub.
+## Anatomy
 
-### Sketch shared library (preferred)
+See the [Sketch library](https://sketch.cloud/s/JaVzz/) page for an overview of the kit before downloading.
 
-For timely updates, install the kit as Sketch shared library. _Note: Be sure you have the latest version of Sketch installed on your machine._
+<AnchorLinks small>
 
-<GridWrapper col_lg="8" flex="true" bleed="true">
-  <ClickableTile
-    title="Add Carbon Design Kit to Sketch library (Beta)"
-    href="sketch://add-library/cloud/JaVzz"
-    type="resource">
+  - [Grid](#grid)
+  - [Symbols](#symbols)
+  - [Text styles](#text-styles)
+  - [Layer styles](#layer-styles)
 
-![](images/sketch-icon.png)
+</AnchorLinks>
 
-  </ClickableTile>
-</GridWrapper>
+### Grid
 
-### Get the kit via Box (IBM internal only)
+The *IBM 2x Grid* is the framework for laying out all visual elements in Carbon.
 
-If you work with a proprietary add-on library and still want to be connected for updates, set up the kit via Box Drive. Using Box Drive sync will ensure that your design kit file will always be up to date, eliminating the need to manually download new versions as they are released.
+<ImageComponent cols="8" caption="2x Grid overview">
 
-1. Follow the instructions on [IBM Digital Design](https://www.ibm.com/standards/web/design-kit/) to get the design kit through Box Drive.
-2. Once you have installed and gained access to the `IBM Design Kit` folder, navigate to the `product design` folder. Select the desired version of Carbon.
-3. When updates are available to the kit library, a purple `Library Updates Available` badge will appear in the top right corner of your Sketch file.
-4. Click `Library Updates Available` then `Accept Updates` in the modal.
+![2x Grid overview](/images/grid-1.png)
 
-<ImageComponent cols="8" caption="Update indicator">
+</ImageComponent>
 
-![Update indicator screenshot](images/designers-5.png)
+All designs should start with the 2x Grid template. Once you have successfully [saved the grid](#gat-the-kit) as a template, navigate to `File → New file from Template` and select the grid template.
 
-</ImageComponent>
+#### Structure
+
+At the core of the 2x Grid concept is the idea of divisions of two. This can be applied to most surfaces, 3-D objects, and architecture as a means to divide space into a helpful grid system. 
 
-### Download the kit from GitHub
+Dividing your space into 2, 4, 8, 16, 32, or 64 columns gives you the basis of the system with the purpose of helping you make decisions and organize your content.
 
-This is the best method for one-time downloads of the kit, or if you are looking for previous versions.
+You can see a full breakdown of the 2x Grid at [IBM.com/design](https://www.ibm.com/design/language/elements/2x-grid).
```

Comment:
Should we be linking out to the IDL grid page or our grid page (included in [layout](https://next.carbondesignsystem.com/guidelines/layout/#2x-grid-fundamentals))? Just an open question

Reply:
Layout page makes more sense, we don't want people to unnecessarily leave our site.